Web/Admin.UI/src/app/components/core/alert-dialog/alert-dialog.component.scss

```
@@ -27,3 +27,18 @@
   display: flex;
   justify-content: center;
 }
+
+::ng-deep .csv-error-label {
```

Stylelint (17.10.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)